cirq/sim/density_matrix_utils_test.py

```diff
@@ -26,6 +26,38 @@ def assert_valid_density_matrix(matrix, num_qubits=1):
                                      dtype=matrix.dtype), matrix)
 
 
+def test_von_neumann_entropy():
+    # 1x1 matrix
+    assert cirq.von_neumann_entropy(np.array([[1]])) == 0
+    # An EPR pair state (|00> + |11>)(<00| + <11|)
+    assert cirq.von_neumann_entropy(
+        np.array([1, 0, 0, 1] * np.array([[1], [0], [0], [1]]))) == 0
+    # Maximally mixed state
+    assert cirq.von_neumann_entropy([[0.5, 0], [0, 0.5]]) == 1
+    # 3x3 state
+    assert np.isclose(
+        cirq.von_neumann_entropy(
+            np.array(
+                np.array([[0.5, 0.5j, 1], [-0.5j, 0.5, 0], [0.7, 0.4, 0.6]]))),
+        1.9, 1e01)
+    # 4X4 state
+    assert np.isclose(
+        cirq.von_neumann_entropy(
+            np.array([[0.5, 0.5j, 1, 3], [-0.5j, 0.5, 0, 4], [0.7, 0.4, 0.6, 5],
+                      [6, 7, 8, 9]])), 52, 1e01)
+    # 2x2 random unitary, each column as a ket, each ket as a density matrix,
+    # linear combination of the two with coefficients 0.1 and 0.9
+    res = cirq.testing.random_unitary(2)
+    first_column = res[:, 0]
+    first_density_matrix = 0.1 * np.outer(first_column, np.conj(first_column))
+    second_column = res[:, 1]
+    second_density_matrix = 0.9 * np.outer(second_column,
+                                           np.conj(second_column))
+    assert np.isclose(
+        cirq.von_neumann_entropy(first_density_matrix + second_density_matrix),
+        0.4689, 1e04)
```

Comment:
Use explicit `atol=` for tolerance arguments.

Reply:
Done.